### PR TITLE
Improves installation doc

### PR DESCRIPTION
**Pull request by crystalin**

### CODE REVIEW

**Comment by crystalin** [comment]
@xlc I was able to make it run, but it needs some documentation also around how to use it as tools like polkadotJs can't be used with it.

**Comment by crystalin** [comment]
I also tried to run it and got:
```
yarn start run-block --endpoint=wss://moonbeam.kaki.dev
2022-10-26 14:40:53        API/INIT: RPC methods not decorated: moon_isBlockFinalized, moon_isTxFinalized
2022-10-26 14:40:53        API/INIT: moonbeam/1803: Not decorating unknown runtime apis: 0xa33d43f58731ad84/2, 0xba8173bf23b2e6f8/1
{"level":30,"time":1666809653364,"pid":29232,"hostname":"CORSAIRONEPRO","endpoint":"wss://moonbeam.kaki.dev","blockHash":"0x5a017006a41a15a2c7997af72256addf1e9e1c51fa8f35d2051a632821285f28","msg":"Args"}
{
  Call: {
    result: '0x0f539f8e84b2425733729e917051462c7fb3c0da30e0a057a8d62620ad773ec9460a84008c43c4317713eecafc215958a59ff1b41db60dfebada06a7449095bb9420370a2f5eda73ba5ec3ef2a1fade21ec908be313ac8d7a3ac4b67d4ed5676d24a8f9a0c066e6d627380ba8dd5b5e1c16cf8a7b09ecba366a62e87f6018be23d7c233c5ba084534a86170672616e6481019ca36d55fbfd6e8ad39b2f756c935483877ef074c8060deaf016cc4572957222dd0106ad91197ea9f33fecb4391e4840ecee4e4531dbd5e10d08f3a60b27c00cf5059e619bb1b05f0507b090cbea40dbee80ddc5057d5f596be3f3003d1dfb0b0466726f6e0903012530f06eedf61dbdfe4e518225c686e889c221c8f1cdec1121a1a3b05943678b1454fde9de8f5ee14d6341bc61fcf62677cc23b1e33cea233e4fec312f46a890f33e1ecd56dc567f759a1d1eb4075d908374d620615c8473392ee2fe9bc8d834bfc60929b247aaad5f5c1433ca1f88da13a8969c6b36ce0eaf0945bbc64b4de0ba5682df0fd2570cfa5e49b2df512cd8aa959874fb4e729b13174583c469284c6aa5716c4f2ae0fd03084abefaa2bc3484392f68ad7af02574896aaac5385c6b17',
    storageDiff: [
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array], [Array], [Array], [Array], [Array], [Array],
      [Array], [Array],
      ... 470 more items
    ]
  }
}
```

File: rust-toolchain [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2022-09-10"
```
Comment (xlc) [inline rust-toolchain:2]: I think we want to put this under executor folder

**Comment by xlc** [comment]
Thanks for remind me to have more docs.
`run-block` only replay a single block and that's it. It should log all he storage changes from that block and you can see we need to tweak the log output format.

`yarn start` or `yarn dev` will run the test node. Defaulting to fork on the latest block of the selected endpoint. And then you can connect to via `localhost:8000` from polkadot apps and start submitting transactions.

**Pull request merged**